apisix/init.lua

```diff
@@ -179,6 +180,8 @@ function _M.http_ssl_phase()
 
     local ok, err = router.router_ssl.match_and_set(api_ctx)
```

Comment:
if reject, is ok equal to `false` ?
if ok equal to `false`, does `ngx_exit(-1)` interrupt the execution of the current request ?

Reply:
It's the original logic and irrelevant to this PR.
On the contrary, the route-level MTLS bypasses the client cert error in the SSL phase, i.e. it relaxes the success restrictions.